main.go

```diff
@@ -703,12 +703,24 @@ func main() {
 	flag.IntVar(&startTimeout, "startTimeout", 10, "Timeout before testing starts")
 	flag.IntVar(&bufferKB, "bufferKB", 64, "TX/RX buffer size in Kilobytes")
 	flag.IntVar(&payloadMB, "payloadMB", 1, "Payload buffer size in Megabytes")
-	flag.StringVar(&serveIP, "serverIP", "0.0.0.0", "IP to listen for requests on")
+	flag.StringVar(&serveIP, "serverIP", "0.0.0.0", "IP to listen for requests on, only needed if two IPs are on the same host. (mostly used for local testing)")
+	flag.StringVar(&hosts, "hosts", "192.168.0.{1...10},10.10.1.{10...20}", "Define servers using an ellipses range. Many ranges can be combined using a comma.")
 	flag.IntVar(&proc, "proc", 32, "Concurrent requests per host")
 	flag.IntVar(&seconds, "seconds", 15, "how long (Seconds) to run hperf")
 	flag.BoolVar(&printAllErrors, "printAll", false, "Print all erros as they happen")
 	flag.Parse()
 
+	splitHosts := strings.Split(hosts, ",")
+	hostList := make([]ellipses.ArgPattern, 0)
+	for _, v := range splitHosts {
```

Comment:
Take an example of ellipses support from warp @zveinn

Reply:
here I guess: https://github.com/minio/warp/blob/master/cli/client.go#L263-L269

Reply:
Done, I did it a bit differently because that code didn't really fit into hperf. But it has the same support now.